test/unit/std/concept/auxiliary_iterator.hpp

```diff
@@ -41,8 +41,6 @@
 
 #include <seqan3/std/iterator>
 
-using namespace seqan3;
-
 using input_iterator               = std::istream_iterator<char>;
 using output_iterator              = ranges::ostream_iterator<char>;
```

Comment:
So that means that the STL has now a `ranges::` namespace that resides outside of `std::`?

Reply:
no, this refers to the ranges namespace from range-v3.